contents/blog/b2b-saas-product-metrics.md

```diff
@@ -0,0 +1,210 @@
+---
+date: 2022-06-14
+title: The most useful B2B SaaS product metrics (and how to measure them)
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/give-back-friday.png
+featuredImageType: full
+categories: ["Guides", "PostHog Academy", "Product analytics"]
+author: ["andy-vandervell"]
+---
+
+So, you're building a B2B SaaS product.
+
+Everyone knows you've got to measure stuff to succeed. This is not news. 
+
+What, why, and how? These are the questions that matter.
+
+In this guide we will:
+
+1. Reveal what makes a useful metric for a B2B product
+2. Explain some of the common metrics used by B2B SaaS companies
+3. Help you choose the right metrics for your product
+
+This guide is about _product_ metrics, rather than revenue or wider business metrics. Naturally, they're important too.
```

Comment:
```suggestion
Please note: this guide is about _product_ metrics, rather than revenue or wider business metrics. Naturally, they're important too.
```